deepchecks/core/check_utils/whole_dataset_drift_utils.py

```diff
@@ -39,7 +40,9 @@ def run_whole_dataset_drift(train_dataframe: pd.DataFrame, test_dataframe: pd.Da
                             random_state: int, test_size: float, n_top_columns: int, min_feature_importance: float,
                             max_num_categories_for_display: int, show_categories_by: str,
                             min_meaningful_drift_score: float,
-                            with_display: bool):
+                            with_display: bool,
+                            dataset_names: Tuple[str] = DEFAULT_DATASET_NAMES
```

Comment:
Oh seems we can. So would try to replace all occurrences of "Train" "Test" with that.